sdk/core/amqp-common/src/errors.ts

```diff
@@ -586,6 +602,11 @@ export function translate(err: AmqpError | Error): MessagingError {
       // not found
       error.retryable = false;
     }
+  } else if (isWebsocketError(err)) {
+    // Translate to a generic Browser specific error for security reasons.
+    // For more information refer to - https://html.spec.whatwg.org/multipage/comms.html#feedback-from-the-protocol
```

Comment:
The link here talks about why the browser doesnt differentiate between the different kinds of "server communication errors". This would be more helpful when we are trying to identify such errors. I would suggest to move it to the docs for `isWebsocketError`. Something like

> Browser doesnt differentiate between the various kinds of service communication errors, therefore look at the error target to identify such category of errors. For more information ....

And then change the comment here to

> // Translate browser communication errors during opening handshake to generic ServiceComminucationError

Reply:
Updated comments and function signature